README.md

```diff
@@ -1,2 +1,36 @@
-# reactive-streams-scalajs
-Reactive Streams bindings for ScalaJS
+# Reactive Streams for Scala.js
+
+The purpose of Reactive Streams is to provide a standard for asynchronous stream processing with non-blocking backpressure.
+
+This repository manages the [Scala.js](http://www.scala-js.org/)
+builds of the API. For details and the TCK see
+[the JVM](https://github.com/reactive-streams/reactive-streams-jvm).
```

Comment:
I'd argue if we're serious about this initiative it must come with a minimal version of the TCK.

Yes, ScalaTest was specifically worked on to allow Scala.JS to be tested, a side effect of that was the asynchronous testing infra, since no blocking allowed.

Not all rules will be able to be verified in this style, but we must provide as much as we can here.